### DISCUSSION TITLE
rotate a lineLayer

### DISCUSSION THREAD

**Discussion by Pueablo** [visgl/deck.gl#7782] [Q&A]
Hello, good morning, I have a question, I would like to rotate a lineLayer around its own point, this is simulating a radar line, I read something about getAngle but it doesn't accept it, I also read about transitions but I can't find how to do it, is there any way to do it? I thank you!

**Comment by Pessimistress** [comment]
There is no `getAngle` in `LineLayer`, please read the [documentation](https://deck.gl/docs/api-reference/layers/line-layer).

You can use the layer with `coordinateSystem: COORDINATE_SYSTEM.METER_OFFSET`, and set `coordinateOrigin` to the point that you want rotate around. See [coordinate systems](https://deck.gl/docs/developer-guide/coordinate-systems#supported-coordinate-systems). For rotation update the [modelMatrix](https://deck.gl/docs/api-reference/core/layer#modelmatrix).

**Reply by Pueablo** [reply]
thank you! I'm going to try it
